cmd/algod/main.go

```diff
@@ -50,10 +50,11 @@ var versionCheck = flag.Bool("v", false, "Display and write current build versio
 var branchCheck = flag.Bool("b", false, "Display the git branch behind the build")
 var channelCheck = flag.Bool("c", false, "Display and release channel behind the build")
 var initAndExit = flag.Bool("x", false, "Initialize the ledger and exit")
+var logToStdout = flag.Bool("o", false, "Write to stdout instead of node.log by overriding config.LogSizeLimit to 0")
```

Comment:
Oh, but now I'm imagining the convention `-o foo` to log to file `foo` and `-o -` for stdout.

Reply:
doc would note "defaults to {data dir}/node.log"

Reply:
I'd rather not add that feature unless there's a real reason that it would be useful. I kinda want to remove most of these flags, not sure they're actually used anymore.

Reply:
I guess picking a different letter would address that concern but all the good letters are taken already.